foyer/forcefields/xml/oplsaa.xml

```diff
@@ -179,8 +179,8 @@
   <Type name="opls_180" class="OS" element="O" mass="15.9994"  def="[O;X2](C)C" desc="O, dialkyl ether" doi="10.1021/ja9621760"/>
   <Type name="opls_181" class="CT" element="C" mass="12.01100" def="[C;X4]([O;%opls_180])(H)(H)H" desc="C(H3OR), methyl ether" overrides="opls_182" doi="10.1021/ja9621760"/>
   <Type name="opls_182" class="CT" element="C" mass="12.01100" def="[C;X4]([O;%opls_180])(H)(H)" desc="C(H2OR), methyl ether" doi="10.1021/ja9621760"/>
-  <Type name="opls_183" class="opls_183" element="C" mass="12.011"/>
-  <Type name="opls_184" class="opls_184" element="C" mass="12.011"/>
+  <Type name="opls_183" class="CT" element="C" mass="12.011" def="[C;X4]([O;%opls_180])(C)(C)H" desc = "Carbon attached to Oxygen in Isopropy ether" doi="10.1021/ja9621760"/>
+  <Type name="opls_184" class="CT" element="C" mass="12.011" def="[C;X4]([O;%opls_180])(C)(C)(C)" desc = "Carbon attached to Oxygen in t-butyl ether" doi="10.1021/ja9621760"/>
```

Comment:
Which Carbon? I presume you mean the central carbon (as in, the one with everything branching off of it) in t-butyl ether:
![image](https://user-images.githubusercontent.com/7935382/73965359-9e59a680-48d9-11ea-9a65-70acddb9ef31.png)

and either in isopropyl ether (I guess this is diisopropyl ether, but think of the one with branching stuff in isopropyl methyl ether):

![image](https://user-images.githubusercontent.com/7935382/73965520-f0023100-48d9-11ea-9595-404ce9a64673.png)

But in principle, the carbons on either side of the oxygen are "Carbon attached to the Oxygen". I don't remember the organic chemistry name for "carbon that everything branches off from"

Reply:
The diisopropyl ether wouldnt match this one, right? Since there are not 3 carbons attached to the carbon which is attached to the oxygen?

Reply:
I think 184 is for the butyl ether only and 183 will match both carbons here